src/main/java/com/mattworzala/debug/client/screens/DebugRenderersGUI.java

```diff
@@ -0,0 +1,94 @@
+package com.mattworzala.debug.client.screens;
+
+import io.github.cottonmc.cotton.gui.client.LightweightGuiDescription;
+import io.github.cottonmc.cotton.gui.widget.WGridPanel;
+import io.github.cottonmc.cotton.gui.widget.WToggleButton;
+import io.github.cottonmc.cotton.gui.widget.data.Insets;
+import net.minecraft.text.LiteralText;
+
+public class DebugRenderersGUI extends LightweightGuiDescription {
+    public DebugRenderersGUI() {
+        WGridPanel root = new WGridPanel();
+        setRootPanel(root);
+        root.setSize(256, 240);
+        root.setInsets(Insets.ROOT_PANEL);
+
+        WToggleButton button = new WToggleButton(new LiteralText("Pathfinders"));
+        button.setToggle(com.mattworzala.debug.DebugRenderer.debugPathfinders);
+        button.setOnToggle(bool -> { System.out.println("Debug: "+bool); com.mattworzala.debug.DebugRenderer.debugPathfinders = bool;});
+
+        WToggleButton button1 = new WToggleButton(new LiteralText("Water"));
+        button1.setToggle(com.mattworzala.debug.DebugRenderer.debugWater);
+        button1.setOnToggle(bool -> { System.out.println("Debug: "+bool); com.mattworzala.debug.DebugRenderer.debugWater = bool;});
+
+        WToggleButton button2 = new WToggleButton(new LiteralText("Height Maps"));
+        button2.setToggle(com.mattworzala.debug.DebugRenderer.debugHeightMap);
+        button2.setOnToggle(bool -> { System.out.println("Debug: "+bool); com.mattworzala.debug.DebugRenderer.debugHeightMap = bool;});
+
+        WToggleButton button3 = new WToggleButton(new LiteralText("Collision"));
+        button3.setToggle(com.mattworzala.debug.DebugRenderer.debugCollision);
+        button3.setOnToggle(bool -> { System.out.println("Debug: "+bool); com.mattworzala.debug.DebugRenderer.debugCollision = bool;});
```

Comment:
These system prints should not be here

Reply:
oh right lol I put those there because it wasn't working and then I saw the issue

Reply:
Fixed it. Pushing changes